heartbeat/status_beater.go

```diff
@@ -100,6 +108,7 @@ func (sb *StatusBeater) PublishEvent(logData []byte, publish func(event beat.Eve
 	}
 	publish(event)
 	logp.Info("heartbeat sent")
+	logp.Debug("Fully Qualified Beatname: %s", fqBeatName)
```

Comment:
We don't want to log the FQBN every time an event is published. The FQBN never changes, so you're just going to be logging the same value over and over again.

Reply:
log the FQBN only in New Function